web/src/routes/(user)/albums/[albumId]/+page.svelte

```diff
@@ -152,6 +152,19 @@
     if (backUrl === AppRoute.SHARING && album.sharedUsers.length === 0) {
       isCreatingSharedAlbum = true;
     }
+
+    if (albumId !== data.album.id) {
+      album = data.album;
+      albumId = data.album.id;
+      currentAlbumName = data.album.albumName;
+      assetStore = new AssetStore({ albumId: album.id });
+      timelineStore = new AssetStore({ isArchived: false }, album.id);
+      if (album.sharedUsers.length > 0) {
+        getFavorite();
+        getNumberOfComments();
+      }
+      refreshAssetGrid = !refreshAssetGrid;
+    }
```

Comment:
This doesn't seem like a "small issue" and I 'm not sure if this actually works. Can you move this into a separate pull request?

Reply:
Aahah sure

Reply:
We use the same trick with [personId]/+page.svelte